states/new_hampshire/elections.toml

```diff
@@ -3,7 +3,52 @@ date = 2020-09-08
 name = "Congressional Primary"
 original_date = 2020-09-08
 
+[20200908.registration]
+in_person_by = 2020-09-08
+sources = ["http://sos.nh.gov/WorkArea/DownloadAsset.aspx?id=8589993085"]
+
+# [20200908.absentee.application]
+# # The Mailed-In or dropped off early Ballot is the Application
```

Comment:
Looks like you need to request being sent a ballot: http://sos.nh.gov/WorkArea/DownloadAsset.aspx?id=8589999336

From: http://sos.nh.gov/VotingEmergency.aspx

Reply:
You find all the things !! ;-)

Reply:
Looks like you found the best source - do you have all the information you need for NH?

Reply:
The key thing in NH this year is that concern about COVID-19 is considered a valid "disability" so you are permitted to use the "Absentee for Disability" forms for registration and for requesting an Absentee Ballot.

This is not a permanent change to the law, just a special allowance for the pandemic.
Normally absentee methods are only permitted for limited reasons.

Reply:
Thanks @jerryneedell. We're doing these dates on an election-by-election basis for that reason. I think we could better convey it though going forwards.